omod/src/main/compass/sass/scheduleAppointment.scss

```diff
@@ -3,7 +3,7 @@
 }
 
 .inlineBox {
-  float: left;
+  display:inline-block;
```

Comment:
Wow thanks for working on this, you have made it more simpler than i had  done cc @dkayiwa  , this LGTM

Reply:
Thanks for your review!

Reply:
```suggestion
  display: inline-block;
```